scripts/coverage.sh

```diff
@@ -1,136 +1,66 @@
 #!/usr/bin/env bash
 #
-# Runs all tests and collects code coverage
+# Run tests and collect code coverage
 #
-# Warning: this process is a little slow
+# Run all:
+#   $ ./script/coverage.sh
 #
+# Run for specific packages
+#   $ ./script/coverage.sh -p solana-account-decoder
+#   $ ./script/coverage.sh -p solana-account-decoder -p solana-accounts-db [-p ...]
 
-if ! command -v grcov; then
-  echo "Error: grcov not found.  Try |cargo install grcov|"
-  exit 1
-fi
-
-if [[ ! "$(grcov --version)" =~ 0.8.[0-9] ]]; then
-  echo Error: Required grcov version not installed
+set -e
+here=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
 
-  echo "Installed version: $(grcov --version)"
+# Check for grcov commands
+if ! command -v grcov >/dev/null 2>&1; then
+  echo "Error: grcov not found.  Try |cargo install grcov|"
   exit 1
 fi
 
-set -e
-cd "$(dirname "$0")/.."
-source ci/_
+# shellcheck source=ci/rust-version.sh
+source "$here/../ci/rust-version.sh" nightly
 
-cargo="$(readlink -f "./cargo")"
+# Clean up
+rm -rf "$here/../target" "$here/../coverage"
+find "$here/.." -type f -name '*.prof*' -exec rm {} +
+find "$here/.." -type f -name '*lcov*' -exec rm {} +
```

Comment:
`-delete` is available for `find`. (i hope this is true for macOS...)

Reply:
yeah, I prefer to use current version to ensure compatibility across different systems!